session.go

```diff
@@ -50,22 +60,15 @@ type Session struct {
 	roTxn *spanner.ReadOnlyTransaction
 }
 
-func NewSession(ctx context.Context, projectId string, instanceId string, databaseId string, clientConfig spanner.ClientConfig, opts ...option.ClientOption) (*Session, error) {
+func NewSession(ctx context.Context, projectId string, instanceId string, databaseId string, opts ...option.ClientOption) (*Session, error) {
 	dbPath := fmt.Sprintf("projects/%s/instances/%s/databases/%s", projectId, instanceId, databaseId)
+	opts = append(opts, defaultClientOpts...)
+
 	client, err := spanner.NewClientWithConfig(ctx, dbPath, clientConfig, opts...)
 	if err != nil {
 		return nil, err
 	}
 
-	if emulatorAddr := os.Getenv("SPANNER_EMULATOR_HOST"); emulatorAddr != "" {
```

Comment:
Now google-cloud-go has emulator hook in admin client initialization.
https://github.com/googleapis/google-cloud-go/blob/master/spanner/admin/database/apiv1/init.go